index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <div>
+        <h1 id="title">Microverse Survey Form</h1>
+        <p id="description">
+            Thank you for taking the time to help us improve our skills
+        </p>
+    </div>
+
+    <form id="survey-form">
```

Comment:
I noticed you didnt close the

tag, please remember as a developer the little things make the difference
I trust you will remember next time
:running_man: :muscle: